app/experimenter/experiments/api/v5/inputs.py

```diff
@@ -12,7 +12,8 @@ class ExperimentInput(graphene.InputObjectType):
 
 class CreateExperimentInput(ExperimentInput):
     name = graphene.String(required=True)
-    slug = graphene.String(required=True)
+    hypothesis = graphene.String(required=True)
+    application = graphene.String(required=True)
```

Comment:
This should be an `enum`, but I can fix this in a later PR as it will require some additional tweaks in my current issue I'm almost done with.

Reply:
@lmorchard if you rebase, my update with `graphene.Enum`'s are in now, so this can be updated to use a `NimbusExperimentApplicationType` enum in a similar manner from my PR. You'd need to make that type like so:
https://github.com/mozilla/experimenter/pull/3803/files#diff-0f23f9b75c80649a9611b784fb9a9c1b2bd38b3bed19cf3bea723fe59763c5a9R17

Add it under the NimbusExperiment:
https://github.com/mozilla/experimenter/pull/3803/files#diff-0f23f9b75c80649a9611b784fb9a9c1b2bd38b3bed19cf3bea723fe59763c5a9R47

Then use it in the Input:
https://github.com/mozilla/experimenter/pull/3803/files#diff-1bc92d6c96b25929bbe8ae9fbccc529d28cdd5a3a276f487b9a99bca09485db3R77

Reply:
I think we should defer this to EXP-462 https://jira.mozilla.com/browse/EXP-462